.github/workflows/codeql-analysis.yml

```diff
@@ -32,9 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [  ]
-        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
-        # Learn more about CodeQL language support at https://git.io/codeql-language-support
+        language: [ 'javascript' ]
```

Comment:
why not golang?

Reply:
There isn't a golang file in this repository.

Reply:
oops, sorry... I though we were on scorecard repo :-)
If there's bash script and dockerfile support, we could enable it

Reply:
actually I'm curious: what does CodeQl check for if we don't tell it which languages to support? Does it detect it automatically? We have only bash script and docker file in this repo.

Reply:
It checks for the GitHub actions. We need a js script file to check for vulnerabilities.

Reply:
OK.